syntax = "proto3";

package testrpc;
option cc_generic_services = true;

service CalculatorService {
  rpc Add(AddRequest) returns (AddResponse);
  rpc Sub(SubRequest) returns (SubResponse);
}

message AddRequest {
  int32 a = 1;
  int32 b = 2;
}

message AddResponse {
  int32 sum = 1;
}

message SubRequest {
  int32 a = 1;
  int32 b = 2;
}

message SubResponse {
  int32 diff = 1;
}

